compiler/rustc_resolve/src/late.rs

```diff
@@ -1103,7 +1126,8 @@ impl<'a: 'ast, 'b, 'ast> LateResolutionVisitor<'a, 'b, 'ast> {
         for i in &mut indices {
             let rib = &self.lifetime_ribs[i];
             let normalized_ident = ident.normalize_to_macros_2_0();
-            if let Some(_) = rib.bindings.get_key_value(&normalized_ident) {
+            if let Some((_, &region)) = rib.bindings.get_key_value(&normalized_ident) {
```

Comment:
```suggestion
            if let Some(region) = rib.bindings.get(&normalized_ident) {
```
Since the key is not used anyway.